ansible/roles/software/catalog/ckan-app/templates/etc/ckan/production.ini.j2

```diff
@@ -204,7 +204,9 @@ ckan.tracking_enabled = True
 set debug = False
 ckan.harvest.mq.type = redis
 ckan.harvest.mq.hostname = localhost
+{% if redis_password is defined %}
```

Comment:
Spoke to @JJediny about this and we should just set the variable in all the ckan based inventories